packages/app/src/App.tsx

```diff
@@ -49,7 +49,7 @@ const app = createApp({
   apis,
   components: {
     SignInPage: props => (
-      <SignInPage {...props} auto provider={githubProvider} />
+      <SignInPage {...props} auto providers={['guest', githubProvider]} />
```

Comment:
Is guest provider read only?

Reply:
Ah, good call, it is definitely not, It looks like adding a read-only capability might be more complicated than I initially thought.  It looks like the backstage demo app gets around this by removing the `register` view from the frontend.

Reply:
I removed this part from this PR so we can land the rest of it.